src/check_datapackage/issue.py

```diff
@@ -31,3 +32,4 @@ class Issue:
     jsonpath: str
     type: str
     message: str
+    instance: Any
```

Comment:
We could also enumerate the possible types here, but I'm not 100% on what they should be. From running

```sh
rg "\"type\": \"" ./src/check_datapackage/schemas/data-package-2-0.json |tr -d " " | tr -d ',' | sort | uniq
```

I see:

```
"type":"array"
"type":"boolean"
"type":"integer"
"type":"number"
"type":"object"
"type":"string"
```

It seems like `array` could be `List`, but what is `object`? Maybe `Dict`?